src/xrpld/rpc/detail/TransactionSign.cpp

```diff
@@ -710,6 +710,108 @@ transactionFormatResultImpl(Transaction::pointer tpTrans, unsigned apiVersion)
 
 //------------------------------------------------------------------------------
 
+static XRPAmount
+getTxFee(Application const& app, Config const& config, Json::Value tx)
```

Comment:
Please make it `[[nodiscard]]`

Reply:
Fixed in a744645f7d2e7142512791a6bb2abaaff26bd62e